internal/cli/atlas/backup/restores_start.go

```diff
@@ -139,11 +130,7 @@ func markRequiredPointInTimeRestoreFlags(cmd *cobra.Command) error {
 		return err
 	}
 
-	if err := cmd.MarkFlagRequired(flag.TargetClusterID); err != nil {
```

Comment:
cluster id is a cloud manager/ops manager thing

Reply:
Is this not the same command that is used for `mongocli`?

Reply:
atlas specific, at some point it was reused for the 3 (or a version of it) and where I believe the mix came from